tests/src/signed_integers/signed_i16_twos_complement/src/main.sw

```diff
@@ -0,0 +1,29 @@
+script;
```

Comment:
I think a script is fine for testing, but optionally you could now be using Sway in-language tests for unit testing libs. The benefit to that is rather than trying to debug exactly which assertion failed in your script, Forc will tell you which test failed. there's also less overhead to get the tests setup, you can just add them directly to the library you want to test, no SDK  or Rust needed.

Reply:
Ref: https://fuellabs.github.io/sway/v0.35.1/book/testing/unit-testing.html

Reply:
Thanks! I'll do that in next PRs if you are ok with that

Reply:
sounds good.